### PR TITLE
Add localization for OpenFolderSchema.json

### PR DESCRIPTION
**Pull request by pieandcakes**
Includes updating strings and a comments file.

### CODE REVIEW

File: loc/LCI/lib/OpenFolderSchema.json.lci [hunk 1]
```diff
@@ -0,0 +1,787 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LCX SchemaVersion="6.0" Name="Z:\src\MIEngine\bin\Desktop.Debug\OpenFolderSchema.json" PsrId="306" FileType="1" SrcCul="en-US" Desc="Commenting file created by LCXAdmin. For more information, please visit http://localizability/longhorn/LcxAdmin.asp" xmlns="http://schemas.microsoft.com/locstudio/2006/6/lcx">
```
Comment (gregg-miskelly) [inline loc/LCI/lib/OpenFolderSchema.json.lci]: >Z:\src\MIEngine\bin\Desktop.Debug [](start = 31, length = 33)

Is this right?

File: src/MIDebugPackage/MIDebugPackage.csproj [hunk 2]
```diff
@@ -95,6 +95,10 @@
     <StartProgram Condition="'$(DevEnvDir)' != ''">$(DevEnvDir)devenv.exe</StartProgram>
     <StartArguments>/rootsuffix Exp</StartArguments>
   </PropertyGroup>
+  <PropertyGroup> 
+  <!-- For Localization -->
+    <EnlistmentRoot>$(MSBuildThisFileDirectory)..\..\</EnlistmentRoot>
```
Comment (gregg-miskelly) [inline src/MIDebugPackage/MIDebugPackage.csproj]: >$(MSBuildThisFileDirectory)..\..\ [](start = 20, length = 33)

We have a property for this already -- $(MIEngineRoot)

File: src/MIDebugPackage/MIDebugPackage.csproj [hunk 3]
```diff
@@ -202,6 +206,19 @@
...
   </ItemGroup>
+  <ItemDefinitionGroup Condition="'$(LocalizationEnabled)'=='true'">
+    <FilesToLocalize>
+      <Languages>$(VS)</Languages>
+      <ProjectFile>$(MSBuildThisFileFullPath)</ProjectFile>
+      <HasLceComments>true</HasLceComments>
+    </FilesToLocalize>
+  </ItemDefinitionGroup>
+  <ItemGroup Condition="'$(LocalizationEnabled)'=='true'">
+    <FilesToLocalize Include="$(OutDir)OpenFolderSchema.json">
```
Comment (gregg-miskelly) [inline src/MIDebugPackage/MIDebugPackage.csproj]: >FilesToLocalize [](start = 5, length = 15)

Do we need to do something to get the resulting files copied somewhere?

File: loc/LCI/lib/OpenFolderSchema.json.lci (hunk 1 above)
Comment (pieandcakes) [inline loc/LCI/lib/OpenFolderSchema.json.lci]: @danyeh Does that filename matter? This was the path on my local machine generated by the LCXAdmin tool

File: src/MIDebugPackage/MIDebugPackage.csproj (hunk 3 above)
Comment (pieandcakes) [inline src/MIDebugPackage/MIDebugPackage.csproj]: @gregg-miskelly The files look like they are put in the bin folder already. There are a couple of steps that I'm still unclear on. I will also need to update the nuget package.

File: loc/LCI/lib/OpenFolderSchema.json.lci (hunk 1 above)
Comment (danyeh) [inline loc/LCI/lib/OpenFolderSchema.json.lci]: @pieandcakes , the path doesn't have impact. As long as the filename is right, the loc toolset will work correctly.

File: src/MIDebugPackage/MIDebugPackage.csproj (hunk 3 above)
Comment (gregg-miskelly) [inline src/MIDebugPackage/MIDebugPackage.csproj]: If you want to merge just this part and do the rest as a subsequent PR that is fine by me. I will approve the PR.

**Pull request merged**